### ISSUE TITLE
incorrect and incomplete instructions in INSTALLATION.md

### ISSUE THREAD

**Issue by adi-uchiha** [PalisadoesFoundation/talawa-admin#508]
**Describe the bug**
1. There are some minor but important missing details in [INSTALLATION.md](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md). The mongodb url do not talks about passing a default db in connection string. If not given the app don't throw a error during `npm run dev` but the connection is not made and the development data base is not created. This is troubling for users new to mongodb. Discussed [here on slack](https://thepalisadoes-dyb6419.slack.com/archives/CSWGQ70UQ/p1677241781770239?thread_ts=1677171179.688719&cid=CSWGQ70UQ)

2. Before [this](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md#i-mongodb-compass) step we have to trigger the creation of user object from register panel. Then only the user object in db will appear. This step is not talked about. 

3. [Here](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/CONTRIBUTING.md#contributing-code) 
> The [organizations.spec.js](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/tests/resolvers/organization_query/organizations.spec.js) test file is a well documented example of what to do for resolver tests.

organizations.spec.js is misplaced I need to fix the path 

Files to be changed : 
[INSTALLATION.md](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md)
[CONTRIBUTING.md](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/CONTRIBUTING.md)

**To Reproduce**

1.
2.
3.
4.

**Expected behavior**

**Actual behavior**
A clear and concise description of how the code performed w.r.t expectations.

**Screenshots**

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by palisadoes** [comment]
Thanks
1. Please use the template and fill in the sections properly. Take the time to get your thoughts together before opening the issue too. It's hard to follow the flow.
1. This issue will fix the admin user problem and the documentation will need to be updated
     - https://github.com/PalisadoesFoundation/talawa-admin/issues/499
1. I updated the java script example link, it was a simple fix. Thanks for pointing it out.
1. The mongo connection string does need to be fixed.
1. Please also open issues to fix one problem at a time.

Though you have a valid point, I'm going to close this issue for some of the reasons above. Please follow the process. It helps everyone.
